legacy/securityconfig_v6/roles.yml

```diff
@@ -17,7 +17,7 @@
 # for the request to be successful. If no role match then the request will be denied. Currently a match must happen within
 # one single role - that means that permissions can not span multiple roles. 
 
-# For <permission>, <indexname or alias> and <type> simple wildcards and regular expressions are possible. 
+# For <permission>, <indexname or alias> and <type> simple wildcards and regular expressions are possible.
```

Comment:
Either don't delete trailing spaces or delete them for the entire file. I'd prefer to handle this in a separate commit/PR.